client-core/src/wallet/default_wallet_client.rs

```diff
@@ -105,161 +104,148 @@ where
     fn new_wallet(
         &self,
         name: &str,
-        passphrase: &SecUtf8,
+        enckey: &SecKey,
```

Comment:
I think `new_wallet` should take `passphrase` and return HMAC.

Reply:
Done